wagtail/admin/views/generic/base.py

```diff
@@ -28,3 +36,60 @@ def get_context_data(self, **kwargs):
         context["page_subtitle"] = self.get_page_subtitle()
         context["header_icon"] = self.get_header_icon()
         return context
+
+
+class BaseOperationView(View):
```

Comment:
Naming this `BaseOperationView` instead of `BaseActionView` as I think the latter could cause confusion with the `action-` stuff recently added to the generic views. That one is inspired by how page create/edit views handle the different actions (e.g. `action-publish`, `action-submit`, etc), which will come into play later when workflows is made generic.

(Maybe in the future we can unify those actions stuff with this base operation view, but I think for now it's better to keep this separate.)